ironic.conf.j2

```diff
@@ -10,7 +10,7 @@ default_boot_interface = ipxe
 default_deploy_interface = direct
 default_inspect_interface = inspector
 default_network_interface = noop
-enabled_bios_interfaces = idrac-wsman,no-bios
+enabled_bios_interfaces = idrac-wsman,no-bios,redfish,idrac-redfish,irmc
```

Comment:
irmc probably needs a separate patch?
if you want to include it here we should probably adapt the commit message

Reply:
Updated commit message.  This was changed upstream based on comment to include all drivers but I forgot to use same commit message.